recipes/r-duckdb/build.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+export DISABLE_AUTOBREW=1
+
+# R refuses to build packages that mark themselves as Priority: Recommended
+mv DESCRIPTION DESCRIPTION.old
+grep -va '^Priority: ' DESCRIPTION.old > DESCRIPTION
+
+export MACOSX_VERSION_MIN="10.13"
```

Comment:
You probably need https://conda-forge.org/docs/maintainer/knowledge_base.html?highlight=macosx_sdk_version#requiring-newer-macos-sdks

Reply:
Thanks, I don't know how I missed that in the documentation :facepalm: